pkg/controller/storagecluster/platform_detection.go

```diff
@@ -18,6 +18,12 @@ var AvoidObjectStorePlatforms = []configv1.PlatformType{
 	configv1.AzurePlatformType,
 }
 
+var TuneFastPlatforms = []configv1.PlatformType{
```

Comment:
I think you may have wanted to add a comment here instead of in the other file?

Reply:
yes